.azure-pipelines-templates/build-ci-container.yml

```diff
@@ -0,0 +1,13 @@
+steps:
```

Comment:
Yeah from a not duplicating stuff perspective I think it would be nice to have a single step which builds images based on the current revision, which can be expanded as needed for different images/destinations.

There's nothing really that says it has to be GitHub actions, but given we have one already and I think github actions are a little nicer and it's the direction it seemed people in the team wanted to move in, I'd vote for doing it via the action

Reply:
Since attestation container will be in an independent repository, we'll leave as it is